src/com/stuypulse/stuylib/network/limelight/LimelightTable.java

```diff
@@ -74,57 +77,57 @@ public LimelightTable(String tableName) {
     /****************************************************************/
 
     // Whether the limelight has any valid targets (0 or 1)
-    public final NetworkTableEntry validTarget;
+    public final DoubleEntry validTarget;
```

Comment:
make sure you check with the limelight docs that they have not changed

Reply:
They're still the same.